CMakeLists.txt

```diff
@@ -22,7 +22,7 @@ message(STATUS "CMake build configuration for micm(${CMAKE_BUILD_TYPE}) ${PROJEC
 include(cmake/StaticAnalyzers.cmake)
 
 # C++ options
-set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD 20)
```

Comment:
If you can, please remove this line. That sets the standard for all translation units and we only really need that on micm. I think I added this, but preferably we would set this only on the micm target in the src/CMakeLists.txt file: `target_compile_features(micm INTERFACE cxx_std_20)`

Reply:
done!

Reply:
Is there a down effect to add this line here? Just try to learn more about CMake.

Reply:
Not necessarily bad to set `CMAKE_CXX_STANDARD`, but cmake is moving more towards things that set options on specific targets rather than doing something system wide.

Reply:
Thanks and glad to know it!